mw-webapp/src/App.tsx

```diff
@@ -1,82 +1,30 @@
-import {useEffect, useState} from "react";
-import {useNavigate} from "react-router-dom";
-import {Header} from "src/component/header/Header";
-import {UserPreviewDAL} from "src/dataAccessLogic/UserPreviewDAL";
-import {FirebaseAnalytics} from "src/FirebaseAnalytics";
+import {useState} from "react";
+import {RouterProvider} from "react-router-dom";
 import {
   DEFAULT_NOTIFICATION_SETTINGS,
   globalContext,
 } from "src/GlobalContext";
-import {useErrorHandler} from "src/hooks/useErrorHandler";
 import {UserPreview} from "src/model/businessModelPreview/UserPreview";
-import {pages} from "src/router/pages";
-import {Router} from "src/router/Router";
-import {AuthService} from "src/service/AuthService";
-import {ThemeWorker} from "src/utils/ThemeWorker";
-
-/**
- * Check is current page is home page
- */
-const getIsHomePage = () => pages.home.getPath({}) === location.pathname;
+import {router} from "src/router/Router";
 
 /**
  * App
  */
 export const App = () => {
-  ThemeWorker.loadTheme();
-  useErrorHandler();
   const [user, setUser] = useState<UserPreview | null>(null);
-  const navigate = useNavigate();
-
-  /**
-   * Get default page path
-   */
-  const getDefaultPagePath = (userUid: string | null) => {
-    return userUid
-      ? pages.user.getPath({uuid: userUid})
-      : pages.allWays.getPath({});
-  };
-
-  /**
-   * OnLog in
-   */
-  const onLogIn = async (userUid: string) => {
-    const currentUserPreview = await UserPreviewDAL.getUserPreview(userUid);
-    setUser(currentUserPreview);
-    const defaultPagePath = getDefaultPagePath(userUid);
-
-    if (getIsHomePage()) {
-      navigate(defaultPagePath);
-    }
-  };
-
-  /**
-   * OnLog out
-   */
-  const onLogOut = async () => {
-    setUser(null);
-    const defaultPagePath = getDefaultPagePath(null);
-
-    if (getIsHomePage()) {
-      navigate(defaultPagePath);
-    }
-  };
-
-  useEffect(() => {
-    AuthService.listenAuthStateChange({onLogIn, onLogOut});
-  }, []);
+  const [isInitialization, setIsInitialization] = useState(true);
```

Comment:
isInitialized, setIsInitialized. And let's make it false by default

Reply:
fixed